src/Hpack/Run.hs

```diff
@@ -68,8 +68,13 @@ renderPackage settings alignment existingFieldOrder sectionsFieldOrder Package{.
     dataFiles :: Element
     dataFiles = Field "data-files" (LineSeparatedList packageDataFiles)
 
+    sourceRepository :: [Element]
     sourceRepository = maybe [] (return . renderSourceRepository) packageSourceRepository
 
+    setupDepends :: [Element]
+    setupDepends = maybe [] (return . renderCustomSetup) packageCustomSetup
```

Comment:
On the nitpicking side of things: This should be `customSetup` now, not `setupDepends`